microsoft-dracut-config.spec

```diff
@@ -17,14 +17,16 @@
 
 
 Name:           microsoft-dracut-config
-Version:        0.0.2
+Version:        0.0.3
 Release:        0
 Summary:        Microsoft Dracut config overlay files
 License:        Apache-2.0
 Group:          System/Management
 URL:            https://github.com/SUSE-Enceladus/microsoft-dracut-config
 Source:         %{name}-%{version}.tar.gz
 BuildRequires:  dracut
+Conflicts:      amazon-dracut-config
+Conflicts:      google-dracut-config
```

Comment:
We should do this by using a generic name as provides and the a conflicts with otherproviders see regionServiceClientConfigGCE as an example. I suggest Provides:  dracut-instance-change-config and Conflicts: otherproviders(dracut-instance-change-config)